jfrog-cli/artifactory/commands/buildinfo/publish.go

```diff
@@ -17,7 +20,12 @@ func Publish(buildName, buildNumber string, config *buildinfo.Configuration, art
 		return err
 	}
 
-	buildInfo, err := createBuildInfoFromPartials(buildName, buildNumber, config, artDetails)
+	buildInfo, partials, err := createBuildInfoFromPartials(buildName, buildNumber, config, artDetails)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Why are doing *setBuildInfoPropertiesForArtifacts* are part fo the *build-publish* command?
This actually adds a setProps request to Artifactory for every published build (even if the files had already been tagged). The *build-add-artifacts* command should do tag the artifacts (set the props) by itself.